BE/src/main/resources/data/schema.sql

```diff
@@ -24,7 +24,9 @@ CREATE TABLE items
     detail_type   VARCHAR(32),
     quantity      BIGINT       NOT NULL,
     reward_point  DECIMAL      NOT NULL,
-    image         VARCHAR(64)  NOT NULL,
+    main_url      VARCHAR(255) NOT NULL,
+    side_one      VARCHAR(255) NOT NULL,
+    side_two      VARCHAR(255) NOT NULL,
```

Comment:
이 필드는 무엇인가요?? 아래처럼 스키마에 코멘트 남길 수 있습니다.

```suggestion
    side_one      VARCHAR(255) NOT NULL COMMENT '이런 식으로 추가 할 수 있습니다',
    side_two      VARCHAR(255) NOT NULL,
```

Reply:
3개의 이미지를 각각 저장하는 필드입니다.
`#`으로 주석을 남겼을때 `sql문` 실행시 인식하는 문제가 있어 주석을 제외했는데, 다음부터 설명이 필요할 경우` COMMENT`를 활용해 보도록 하겠습니다.